### DISCUSSION TITLE
Mute Video after Clicking into Post If Muted by Default

### DISCUSSION THREAD

**Discussion by patrickdemers6** [Docile-Alligator/Infinity-For-Reddit#1157] [Completed Ideas] [answered]
In my installation, I have videos muted by default. Sometimes, I will unmute a video, then click into its comments. Once I am finished reading the comments and press the back button, the video begins playing again, with audio. This can be unexpected an unpleasant.

I would love it if when I click into a post, the video is automatically muted. With this in place, the video would be back to muted when I back out of the post. This would only be for users that have Settings -> Video -> Mute Videos enabled.

Thoughts?

**Comment by Wladefant** [comment]
Hey, thanks for your contributions. 
I do not fully understand what you mean. You have your videos by default muted. Right? Then you unmute them, scroll through the comments(no sound) and when tapping the back button  the video is unmuted. Do I understand you correctly because that is not what happens in my case.

**Reply by patrickdemers6** [reply]
Yes, videos are muted by default and I have autoplay on.

My steps are:
- Unmute video in feed that I find interesting.
- Click into comments (video in the post starts from beginning and is muted)
- I scroll comments.
- Go back to feed by pressing back button or swiping out of post.
- The video plays with sound enabled. This is where I would like it to be back to muted.

**Reply by Wladefant** [reply]
@patrickdemers6 can reproduce that now. That does make sense

**Comment by Wladefant** [comment]
It should also work for video>mute autoplaying videos

**Comment by patrickdemers6** [comment, marked as answer]
Addressed by PR #1230.
